extension/background.js

```diff
@@ -1,79 +1,107 @@
-username = "";
+var username = "";
```

Comment:
Either way your whole program will  have access to it. If it needs to be on the window than I can change it to `window.username = "";` instead if you'd like